src/App.vue

```diff
@@ -2,9 +2,21 @@
   <div>
     <TodoHeader @addTodo="addTodo"/>
 
+      <Notodos v-if="!todos.length"/>
+      
+      <TodoDisplay
+        v-else :todos="todos"
+        :showingModal="showingModal"
```

Comment:
boolean variables should start with is prefix, rename showingModal to isModalVisible or something like that